core/primitives/src/stateless_validation/state_witness.rs

```diff
@@ -146,25 +184,9 @@ pub struct ChunkStateWitness {
     /// After these are applied as well, we should arrive at the pre-state-root
     /// of the chunk that this witness is for.
     pub implicit_transitions: Vec<ChunkStateTransition>,
-    #[deprecated(
```

Comment:
`new_transactions` will be "undeprecated" soon: https://github.com/near/nearcore-private/pull/126